causalml/inference/nn/cevae.py

```diff
@@ -0,0 +1,118 @@
+"""
+This module calls the CEVAE[1] function implemented by pyro team. CEVAE demonstrates a number of innovations including:
+
+- A generative model for causal effect inference with hidden confounders;
+- A model and guide with twin neural nets to allow imbalanced treatment; and
+- A custom training loss that includes both ELBO terms and extra terms needed to train the guide to be able to answer
+counterfactual queries.
+
+Generative model for a causal model with latent confounder z and binary treatment w:
+        z ~ p(z)      # latent confounder
+        x ~ p(x|z)    # partial noisy observation of z
+        w ~ p(w|z)    # treatment, whose application is biased by z
+        y ~ p(y|t,z)  # outcome
+Each of these distributions is defined by a neural network.  The y distribution is defined by a disjoint pair of neural
+networks defining p(y|t=0,z) and p(y|t=1,z); this allows highly imbalanced treatment.
+
+**References**
+
+[1] C. Louizos, U. Shalit, J. Mooij, D. Sontag, R. Zemel, M. Welling (2017).
+    | Causal Effect Inference with Deep Latent-Variable Models.
+    | http://papers.nips.cc/paper/7223-causal-effect-inference-with-deep-latent-variable-models.pdf
+    | https://github.com/AMLab-Amsterdam/CEVAE
+"""
+
+import logging
+import torch
+from pyro.contrib.cevae import CEVAE
```

Comment:
Let's import `CEVAE` as `CEVAEModel` and rename `CEVAE_MODDEL` class as `CEVAE`.

Reply:
Thanks! Updated now